Crash on `clear` when looking into scrollback buffer (#3114)
jwilm/alacritty: CHANGELOG.md

- Performance bottleneck when clearing colored rows
- Vague startup crash messages on Windows with WinPTY backend
- Deadlock on Windows when closing Alacritty using the title bar "X" button (ConPTY backend)

> - Crash on `clear` when looking into scrollback buffer

Reviewer: `looking into scrollback buffer` might be a bit confusing, maybe the following is a bit better?

```suggestion
- Crash on `clear` when scrolled up in history
```
